src/device-state/target-state.ts

```diff
@@ -85,33 +93,42 @@ export const update = async (
 		const endpoint = url.resolve(apiEndpoint, `/device/v2/${uuid}/state`);
 		const request = await getRequestInstance();
 
-		const params: Headers = {
-			json: true,
+		const params: any = {
+			headers: {
+				Authorization: `Bearer ${currentApiKey}`,
+				json: true,
```

Comment:
This isn't a header, it's a top level param to `request` that means we request a json response and will automatically parse it as well